pootle/urls.py

```diff
@@ -30,7 +31,15 @@
     url(r'^accounts/evernote/', include('evernote_auth.urls')),
     url(r'^admin/reports/', include('evernote_reports.urls')),
     url(r'', include('evernote_reports.profile_urls')),
+)
+
+if settings.POOTLE_ENABLE_OFFLINE:
```

Comment:
With that said I agree the POOTLE_ENABLE_OFFLINE setting is redundant if the app can be easily dropped from installed_apps.